docs/guides/cloud/integrations/slack.mdx

```diff
@@ -231,3 +146,22 @@ will be able to install the Cypress integration for Slack again in the future.
 2. Within the Slack Integration, click **Configure**.
 3. Click **Uninstall Slack Integration** to uninstall the Cypress integration
    for Slack.
+
+## Security and compliance
+
+Review our [Privacy Policy](https://on.cypress.io/privacy-policy) for information about how the Cypress app for Slack collects and
+manages your information.
+
+### Authorizations
+
+When authorized, Cypress can:
+
+- View information about a user’s identity
+- Post messages to specific channels in Slack & direct message conversations
+- Show previews of cloud.cypress.io URLs in messages
+- View URLs from cloud.cypress.io
```

Comment:
What? I don't understand this. We are authorized to see what data from the user's slack here? We already have urls from our own cloud account...